inc/cleantalk-updater.php

```diff
@@ -1275,3 +1275,11 @@ function apbct_update_to_6_17_2()
     $cron->removeTask('clear_old_session_data');
     $cron->addTask('clear_old_session_data', 'apbct_cron_clear_old_session_data', 86400);
 }
+
+function apbct_update_to_6_38()
```

Comment:
As I remember, no need to write updater for the new options. Not is?

Reply:
Fixed in new commits